### ISSUE TITLE
Crashes on missing author information

### ISSUE THREAD

**Issue by Jaliborc** [ButlerLogic/action-autotag#1]
The action keeps crashing on line 74 for me, when it tries to access `commit.author.login`. Can you make that variable optional?

**Comment by coreybutler** [comment]
Ah, sorry. Yes, that should be optional. I'll close this when the fix is made.

**Comment by coreybutler** [comment]
This was a quick fix and is available in the master branch. You can test using the following:

```yaml
name: My Workflow

on: 
  push:
    branches:
    - master

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - uses: butlerlogic/action-autotag@master             <------------- use master
      with:
        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Once another issue is confirmed to be resolved, I'll release v1.0.3.

**Comment by Jaliborc** [comment]
Ok, thanks. Will try it out.
